README.md

```diff
@@ -60,22 +60,46 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension supports copybooks called by your source code that are stored in a local folder in your workspace. If your copybooks are stored in mainframe data sets, you can use a Zowe CLI z/OSMF profile to automatically download them from the mainframe to your workspace. 
 
-#### Retrieving Copybooks
+To enable copybook support, you specify the local folders and mainframe data sets that contain copybooks used in your project in the workspace settings. When a copybook is called in the program, the folders and data sets are searched in the order they are listed for files and members that match the copybook's name. If a copybook with the same file name is located in both a local folder and a mainframe data set, the one in the local folder is used.
 
-To set up automatic copybook retrieval from the mainframe, **follow these steps:**
+Ensure that copybook names are called in your source code in upper case, for example `COPY BOOK1` and not `COPY book1`. Lower case copybook names are not resolved by the extension and are marked as errors.
```

Comment:
Not exactly, copybook names in a copy statement are case insensitive. The user should make sure that the file name in the folder is in the upper case, or it may be not resolved.

Reply:
Thanks, I've just tested it and this is exactly what happens. Fixed in latest commit.